docs/admin-holder/0.1/README.md

```diff
@@ -645,6 +666,33 @@ Example:
 
 `comment` (String; Optional): Optional comment.
 
+### presentation-request-reject
+
+Reject presentation request.
+
+Example:
+
+```json
+{
+  "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/admin-holder/0.1/presentation-request-reject",
+  "@id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+  "presentation_exchange_id": presentation_request_received[
+    "presentation_exchange_id"
+  ],
+  "message_description": msg_desc,
+}
+```
+
+#### Fields
+
+`@type` (String): Message type
+
+`@id` (String; Optional): Message identifier
+
+`presentation_exchange_id` (String): Presentation to match credentials to.
```

Comment:
The description here should be more along the lines of "presentation to reject" or similar.